### PR TITLE
[AL-54] Add information model

### PR DESCRIPTION
**Pull request by pvnamGit**
## Jira link
[AL-54](https://capstone-alumni.atlassian.net/jira/software/c/projects/AL/boards/1?modal=detail&selectedIssue=AL-54)

## Describe your changes
Add new model Information map to 

## Checklist before requesting a review
1. [x] I have performed a self-review of my code
2. [x] Have you lint your code locally before submission?

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -68,19 +75,21 @@ model User {
   accessStatus  AccessStatus @default(PENDING) @map("access_status")
   accessMode    AccessMode   @default(FULL_ACCESS) @map("access_mode")
 
-  class Class? @relation(fields: [classId], references: [id], onDelete: SetNull)
+  class       Class?        @relation(fields: [classId], references: [id], onDelete: SetNull)
+  information Information[]
```
Comment (congtrung6391) [inline prisma/schema.prisma:79]: 1-1 relation https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-one-relations

File: prisma/schema.prisma [hunk 2]
```diff
@@ -68,19 +75,21 @@ model User {
...
-  name      String
-  classes   Class[]
+  id          String        @id @default(cuid())
+  archived    Boolean       @default(false)
+  createdAt   DateTime      @default(now()) @map("created_at")
+  updatedAt   DateTime      @default(now()) @updatedAt @map("updated_at")
+  code        String        @unique
+  name        String
+  classes     Class[]
+  information Information[]
```
Comment (congtrung6391) [inline prisma/schema.prisma:92]: why grade has information?

File: prisma/schema.prisma [hunk 3]
```diff
@@ -95,7 +104,35 @@ model Class {
   description String?
   users       User[]
 
-  grade Grade @relation(fields: [gradeId], references: [id])
+  grade       Grade         @relation(fields: [gradeId], references: [id])
+  information Information[]
```
Comment (congtrung6391) [inline prisma/schema.prisma:108]: same as above?

File: prisma/schema.prisma [hunk 4]
```diff
@@ -95,7 +104,35 @@ model Class {
...
+  gradeId              String         @map("grade_id")
+  gradeCode            String         @map("grade_code")
+  gradeName            String         @map("grade_name")
+  classId              String         @map("class_id")
+  className            String         @map("class_name")
+  userId               String         @unique
+
+  user  User  @relation(fields: [userId], references: [id])
+  grade Grade @relation(fields: [gradeId], references: [id])
+  class Class @relation(fields: [classId], references: [id])
```
Comment (congtrung6391) [inline prisma/schema.prisma:135]: should not duplicate this, just keep grade, class in User table

File: prisma/schema.prisma [hunk 5]
```diff
@@ -95,7 +104,35 @@ model Class {
...
+  grade       Grade         @relation(fields: [gradeId], references: [id])
+  information Information[]
 
   @@map("classes")
 }
+
+model Information {
+  id                   String         @id @default(cuid())
+  bio                  String
+  userFullName         String         @map("user_full_name")
```
Comment (congtrung6391) [inline prisma/schema.prisma:116]: use `name ` in User only

Comment (pvnamGit) [reply prisma/schema.prisma]: updated

File: prisma/schema.prisma (hunk 1 above)
Comment (pvnamGit) [reply prisma/schema.prisma]: updated

File: prisma/schema.prisma [hunk 6]
```diff
@@ -99,3 +108,27 @@ model Class {
 
   @@map("classes")
 }
+
+model Information {
+  id                   String         @id @default(cuid())
+  bio                  String
+  name                 String
```
Comment (congtrung6391) [inline prisma/schema.prisma:115]: I mean remove the name of information

File: prisma/schema.prisma [hunk 7]
```diff
@@ -99,3 +108,27 @@ model Class {
...
+  avatarUrl            String         @map("avatar_url")
```
Comment (congtrung6391) [inline prisma/schema.prisma:116]: add `coverImageUrl `for background image in profile

File: prisma/schema.prisma (hunk 6 above)
Comment (pvnamGit) [reply prisma/schema.prisma:115]: ok

File: prisma/schema.prisma (hunk 7 above)
Comment (pvnamGit) [reply prisma/schema.prisma:116]: added

**Review by congtrung6391 — APPROVED** [review]

**Pull request merged**